mycore-mods/src/main/java/org/mycore/mods/enrichment/MCRDataSourceFactory.java

```diff
@@ -32,42 +32,39 @@
  * Per data source, for each identifier type, there must be a pattern configured
  * that defines the URI to get the data for this type of identifier
  *
+ * @author Frank Lützenkirchen
  * @see MCRIdentifierResolver
- * 
- * As a global parameter, or per data source, it can be configured whether 
+ * <p>
```

Comment:
Is this intended - where is the closing <p>?

Reply:
Changes to JacaDoc were unintentional. Will undo. That being said, closing `<p>`-tags are optional in HTML and especially discouraged in the 3.2 version of HTML used by JavaDoc.